.github/workflows/functional-tests-approval.yaml

```diff
@@ -0,0 +1,17 @@
+name: 'Functional test approval using environments'
```

Comment:
Can we simplify this name? This is going to appear in the UI for users. This is the name I chose when we were experimenting to make it distinct.